packages/boa/lib/proxy.js

```diff
@@ -331,8 +331,8 @@ function _internalWrap(T, src={}) {
   });
 
   // Create the proxy object for handlers
-  let newTarget;
-  return (newTarget = new Proxy(src, {
```

Comment:
Unnecessary changes.

Reply:
This is for clarity.

Reply:
But I don't see it's clarity more, and it's unrelated to this PR.